designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,47 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+Removing the tab ‘Virtual Machine Templates’ from the side navigation and combining it with ‘Virtual Machines’, so both are located under ‘Virtualization’.
+This PR will focus on how users can create a VM when they choose a to create a VM from a template.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual machine templates tab where they can see the full list of templates.
+From this point, we suggest 2 options to create a VM:
+
+- Create template via an action from the kebab menu
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+- Create template via an action from the kebab menu and also allow an action link
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
+
+Both options will lead the user to the first step of the wizard where they will need to provide a name for the VM they want to create.
+
+![UserNeedsToProvideName](img/provide-name.png)
```

Comment:
I think they'd just be dropped into the general step here right? Otherwise we are creating a substep for the first step.

Reply:
yes, it makes sense but then it makes the 1st step here to be 'General' and it's different from the VM tab left nav